eclipse-theia/theia
# vsx: display error message when failing to fetch

#### What it does


The pull-request updates the **Extensions** view to display an error alert message when we fail to fetch extensions from the marketplace, like we had during the week. The changes should help with the user-experience so they have feedback when something goes wrong.

_Open-VSX being down:_

![Screen Shot 2022-07-22 at 2 12 09 PM](https://user-images.githubusercontent.com/40359487/180499104-4c5247b6-af6a-4369-aa92-d212abd005e9.png)

#### How to test


1. set the `VSX_REGISTRY_URL` environment variable to some fake url (ex: `https://open-vsx.foo`)
2. start the application.
3. open the **Extensions** view.
4. confirm the error message is visible when searching and failing to fetch extensions.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
